migration/i7.1z/postgresql/202006181620_IDEMPIERE-4339.sql

```diff
@@ -0,0 +1,95 @@
+;
+
+-- Jun 18, 2020, 4:14:02 PM CEST
+UPDATE AD_Field SET SeqNo=90, AD_Reference_Value_ID=NULL, AD_Val_Rule_ID=NULL, IsToolbarButton=NULL,Updated=TO_TIMESTAMP('2020-06-18 16:14:02','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=0 WHERE AD_Field_ID=56648
+;
+
+-- Jun 18, 2020, 4:14:02 PM CEST
+UPDATE AD_Field SET SeqNo=100, AD_Reference_Value_ID=NULL, AD_Val_Rule_ID=NULL, IsToolbarButton=NULL,Updated=TO_TIMESTAMP('2020-06-18 16:14:02','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=0 WHERE AD_Field_ID=56647
+;
+
+-- Jun 18, 2020, 4:14:02 PM CEST
+UPDATE AD_Field SET SeqNo=0, AD_Reference_Value_ID=NULL, AD_Val_Rule_ID=NULL, IsToolbarButton=NULL,Updated=TO_TIMESTAMP('2020-06-18 16:14:02','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=0 WHERE AD_Field_ID=205067
+;
+
+-- Jun 18, 2020, 4:14:02 PM CEST
+UPDATE AD_Field SET SeqNo=0, AD_Reference_Value_ID=NULL, AD_Val_Rule_ID=NULL, IsToolbarButton=NULL,Updated=TO_TIMESTAMP('2020-06-18 16:14:02','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=0 WHERE AD_Field_ID=56655
+;
+
+-- Jun 18, 2020, 4:14:19 PM CEST
+UPDATE AD_Tab SET IsSingleRow='N',Updated=TO_TIMESTAMP('2020-06-18 16:14:19','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=0 WHERE AD_Tab_ID=53188
+;
+
+-- Jun 18, 2020, 4:14:24 PM CEST
+UPDATE AD_Tab SET IsSingleRow='N',Updated=TO_TIMESTAMP('2020-06-18 16:14:24','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=0 WHERE AD_Tab_ID=53189
+;
+
+-- Jun 18, 2020, 4:14:36 PM CEST
+UPDATE AD_Tab SET IsSingleRow='N',Updated=TO_TIMESTAMP('2020-06-18 16:14:36','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=0 WHERE AD_Tab_ID=53190
+;
+
+-- Jun 18, 2020, 4:15:37 PM CEST
+UPDATE AD_Column SET AD_Val_Rule_ID=200138, IsToolbarButton='N',Updated=TO_TIMESTAMP('2020-06-18 16:15:37','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=0 WHERE AD_Column_ID=56755
+;
+
+-- Jun 18, 2020, 4:15:52 PM CEST
+UPDATE AD_Column SET AD_Val_Rule_ID=200139, IsToolbarButton='N',Updated=TO_TIMESTAMP('2020-06-18 16:15:52','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=0 WHERE AD_Column_ID=56757
+;
+
+-- Jun 18, 2020, 4:15:52 PM CEST
+UPDATE AD_Tab SET IsSingleRow='N',Updated=TO_TIMESTAMP('2020-06-18 16:14:36','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=0 WHERE AD_Tab_ID=53191
+;
+
+-- Jun 24, 2020, 5:06:03 PM CEST
+INSERT INTO AD_Process (AD_Process_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,Name,Description,IsReport,Value,IsDirectPrint,Classname,AccessLevel,EntityType,Statistic_Count,Statistic_Seconds,IsBetaFunctionality,IsServerProcess,ShowHelp,CopyFromProcess,AD_Process_UU,AllowMultipleExecution,XXA_IsDossierAction) VALUES (200119,0,0,'Y',TO_TIMESTAMP('2020-06-24 17:06:02','YYYY-MM-DD HH24:MI:SS'),0,TO_TIMESTAMP('2020-06-24 17:06:02','YYYY-MM-DD HH24:MI:SS'),0,'Insert Parameters','This process will add required parameters for current web service type','N','WebServiceTypeCreateParameters','N','org.idempiere.webservices.process.WebServiceTypeCreateParameters','6','D',0,0,'N','N','S','N','2b94e537-3bb4-4e25-8277-13c16061e3a6','P','N')
```

Comment:
@nmicoud - I tried to apply this migration script and it failed because of the column XXA_IsDossierAction (same in oracle).